tutorials/shaders/compute_shaders.rst

```diff
@@ -183,7 +183,7 @@ The steps we need to do to compute our result are:
     var computeList = rd.ComputeListBegin();
     rd.ComputeListBindComputePipeline(computeList, pipeline);
     rd.ComputeListBindUniformSet(computeList, uniformSet, 0);
-    rd.ComputeListDispatch(computeList, xGroups: 5, yGroups: 1, zGroups: 1);
+    rd.ComputeListDispatch(computeList, xGroups: 2, yGroups: 1, zGroups: 1);
```

Comment:
Please read the paragraph under this code block, it explains why we use 5 groups